agent/pom.xml

```diff
@@ -70,7 +70,7 @@
                     <Agent-Class>com.lightstep.opentelemetry.agent.LightstepBootstrap</Agent-Class>
                     <Premain-Class>com.lightstep.opentelemetry.agent.LightstepBootstrap
                     </Premain-Class>
-                    <Implementation-Version>1.5.3</Implementation-Version>
+                    <Implementation-Version>1.6.0</Implementation-Version>
```

Comment:
One thing I've wanted to do is to simply re-use here `<opentelemetry.instrumentation.version>` but I wonder if there was a reason why Sergey (previous maintainer) did this. Maybe let's dig into that in a follow up.